src/styles/main.scss

```diff
@@ -1,7 +1,139 @@
 @import "utils";
 @import "fonts";
 @import "typography";
+@import "header";
+@import "nav";
+@import "info";
+@import "gallery";
+@import "consultation";
 
-body {
-  background: $c-gray;
+* {
+  box-sizing: border-box;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+ul,
+li,
+button,
+input,
+textarea {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  line-height: 27px;
+  color: #3e3e3e;
+  list-style: none;
```

Comment:
I dont think you need specify all this styles to each tag

Reply:
in the process

Reply:
fixed